src/control/server/storage/scm/provider.go

```diff
@@ -602,6 +609,43 @@ func (p *Provider) clearMount(req FormatRequest) error {
 	return nil
 }
 
+// makeMountPath will build the target path by creating any non-existent
+// subdirectories and setting ownership to target uid/gid to ensure that the
+// target user is able to access the created mount point if multiple layers
+// deep. If subdirectories in path already exist, their permissions will not be
+// modified.
+func (p *Provider) makeMountPath(req FormatRequest) error {
+	path := req.Mountpoint
+
+	if !filepath.IsAbs(path) {
+		return errors.Errorf("expecting absolute target path, got %q", path)
+	}
```

Comment:
Why not just resolve the path to an absolute path using `filepath.Abs(req.Mountpoint)`? Is there some downside?

Reply:
I think it would be a bit weird to specify a relative path in the config file for a SCM mount, do you think this is a real use case? I would be inclined to only support absolute path in config for SCM mount dir to avoid creating mounts in unexpected locations e.g. /usr/bin.